steps/elasticsearch/session.go

```diff
@@ -36,39 +33,42 @@ type Session struct {
 	SearchResult     golium.Map
 	Correlator       string
 	indexedDocuments []*IndexedDocument
+	ESServiceClient  ClientFunctions
 }
 
 // ConfigureClient creates a elasticsearch connection based on the URI.
 func (s *Session) ConfigureClient(ctx context.Context, config elasticsearch.Config) error {
 	var err error
-	if s.Client, err = elasticsearch.NewClient(config); err != nil {
+	if s.Client, err = s.ESServiceClient.ElasticsearchNewClient(config); err != nil {
```

Comment:
Remove `Elasticsearch` from `ElasticsearchNewClient` because is placed on service name 👍🏽